addons/account/models/account_account.py

```diff
@@ -438,7 +438,11 @@ def _compute_internal_group(self):
     @api.depends('account_type')
     def _compute_reconcile(self):
         for account in self:
-            account.reconcile = account.account_type in ('asset_receivable', 'liability_payable')
+            if account.internal_group in ('income', 'expense', 'equity'):
+                account.reconcile = False
+            elif account.account_type in ('asset_receivable', 'liability_payable'):
+                account.reconcile = True
+            # For other asset/liability accounts, don't do any change to account.reconcile.
 
```

Comment:
I think we could do that in oneline with an "and" but it's maybe better in terms of visibility like that :smile: